lib/abide_dev_utils/xccdf.rb

```diff
@@ -10,14 +10,25 @@
 module AbideDevUtils
   # Contains modules and classes for working with XCCDF files
   module XCCDF
+    # Generate map for CEM
+    def self.gen_map(xccdf_file, opts)
+      type = opts.fetch(:type, 'cis')
+      case type.downcase
+      when 'cis'
+        new_map = Benchmark.new(xccdf_file).gen_map(**opts)
+      else
+        raise AbideDevUtils::Errors::ProfileUnsupportedError
```

Comment:
See my comment on `lib/abide_dev_utils/xccdf.rb`

Reply:
Does a case statement have a shorthand version?

Reply:
Nope, case statements only have the syntax you're using.

Reply:
I think this error should be `AbideDevUtils::Errors::UnsupportedXCCDFError`. Also, errors should have error messages with them.